helpers/wait_for_documentserver_start.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+puts 'Sleep for waiting documentserver start'
+waiting_time = 90 # Timer limit for documentserver
+def seconds_left(param)
+  while param.positive?
```

Comment:
Simpler variant
```
waiting_time.times do |i|
puts("waiting for #{i} seconids")
sleep 1
end
```

I will not extract this to method, it's simple enought in plain text